EXOSIMS/Prototypes/StarCatalog.py

```diff
@@ -63,11 +63,11 @@ class StarCatalog(object):
         coords (astropy.coordinates.SkyCoord):
             SkyCoord object (ICRS frame) containing right ascension, declination, and
             distance to star in units of deg, deg, and pc
-        pmra (~astropy.units.Quantity(~numpy.ndarray(float))):
+        pm_ra_cosdec (~astropy.units.Quantity(~numpy.ndarray(float))):

```

Comment:
These need to be removed entirely from the docstring as they're no longer attributes of the StarCatalog object